packages/mail/script/sendMailsToSegment/cleanedUserSubscriptionInvitation.js

```diff
@@ -0,0 +1,68 @@
+const yargs = require('yargs')
+const PgDb = require('@orbiting/backend-modules-base/lib/PgDb')
+
+const sendMailsToSegment = require('./sendMailsToSegment')
+
+const argv = yargs
+  .option('dry-run', {
+    default: true,
+  })
+  .option('limit', {
+    default: 1000,
+  })
+  .help()
+  .version().argv
+
+PgDb.connect().then(async (pgdb) => {
+  if (argv.dryRun) {
```

Comment:
This flag is not doing what description says; it will exit before able to schedule email (instead of sending).